superset/utils/cache_manager.py

```diff
@@ -40,27 +38,27 @@ def _init_cache(
     ) -> None:
         cache_config = app.config[cache_config_key]
         cache_type = cache_config.get("CACHE_TYPE")
-        if app.debug and cache_type is None:
-            cache_threshold = cache_config.get("CACHE_THRESHOLD", math.inf)
+        if required and cache_type in (None, "SupersetCache"):
+            if cache_type is None:
+                logger.warning(
+                    "Falling back to built-in key-value cache for the following "
```

Comment:
Maybe "Falling back to the built-in cache, that stores data in the metadata database, for the following..."

Reply:
I like that 👍